### ISSUE TITLE
naming conventions

### ISSUE THREAD

**Issue by RenMan2** [arrowtype/shantell-sans-specimen#9]
I installed Shantell Sans, and unlike other large-family typefaces, the light, medium, semi-bold, and extra-bold did not show up in my word processing program. Even with only a third of the fonts (R/RI/B/BI) it was a welcome addition and worlds ahead of Comic Sans. Now it we could only fix the naming conventions, so I could enjoy the rest of the fonts….

**Comment by arrowtype** [comment]
Hi @RenMan2, thanks for the kind words, and thanks for taking time to file an issue to let me know about this!

I don't have a real answer for you currently, but could you please re-file this issue, along with the specs of your OS and apps (including versions) in the main Shantell Sans repo? I'm not sure when I might be able to look into this further, but that will help me to track the issue, and help others find it if they experience the same thing.

http://github.com/arrowtype/shantell-sans/issues

**Comment by RenMan2** [comment]
I have a Dell laptop i7, 32GB RAM, running the latest version of Windows
11. I'm using WordPerfect 21.0.0.1994
I installed Shantell Sans, and unlike other large-family typefaces, the
light, medium, semi-bold, and extra-bold did not show up in my word
processing program.

In the past, when I've had this issue, the problem has been naming
conventions. When everything is working correctly, The R/B/RI/BI just shows
up as Fontname, and other weights show up as Fontname-thin, Fontmane-medium
etc. Shantell Sans with all 18 fonts installed just shows up as "Shontell
Sans Normal"

I reposted on GitHub as you asked.

[image: font family.jpg]
